### PR TITLE
Added a try/except statement to main.py function add_book to avoid bo…

### PR DESCRIPTION
**Pull request by DashKazak**
#11 issue addressed and fixed.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -33,8 +35,12 @@ def create_menu():
...
 def add_book():
     new_book = ui.get_book_info()
-    new_book.save()
-    
+    try: 
+        new_book.save() #the program will try to save the file
+        # but, in bookstore.py line 107 we indicated that the book list can't have duplicates. If this error is encountered, the program will jump to the except statement below instead of printing long developer log error message. 
+        
+    except: 
+        print(f'This book is already on your list - {new_book}')
```
Comment (claraj) [inline main.py]: notice there are no other print statements in main.py. How does main show messages to the user? Recommend the same process for this message.

**Comment by wyatt-geiger** [comment]
Hey @DashKazak I went ahead and put what I believe to be the solution in our Slack channel since Clara mentioned there shouldn't be a print statement.

**Comment by DashKazak** [comment]
I updated the code to match the requirements -- please review

**Comment by wyatt-geiger** [comment]
LGTM

**Pull request merged**